### DISCUSSION TITLE
Migration to docker compose v2?

### DISCUSSION THREAD

**Discussion by sungminlee114** [shellhub-io/shellhub#2692] [Ideas]
Hi, thanks for the great project.

[Docker compose v1 (`docker-compose`) will be removed by end of June 2023](https://docs.docker.com/compose/compose-v2/) and I suggest migrating to Docker compose v2 (`docker compose`).

**Comment by gustavosbarreto** [comment]
We are already using the V2 spec, which is supported by the standalone binary built in Go (`docker-compose`). This binary will continue to exist, but the old Python binary that does not support V2 will be discontinued. All our scripts depend on `docker-compose`, and it does seem like a good idea to start using the `docker compose` CLI plugin on systems that do not have the standalone binary.

I agree with your suggestion, thanks for pointing out.

**Comment by gustavosbarreto** [comment]
I've created a new issue to track this.

https://github.com/shellhub-io/shellhub/issues/2744
